platform/core/src/services/StudyPrefetcherService/StudyPrefetcherService.ts

```diff
@@ -0,0 +1,495 @@
+import { cache, imageLoadPoolManager, imageLoader, Enums } from '@cornerstonejs/core';
```

Comment:
This shouldn't be here, if the study prefetcher relies on cornerstone, the service should be inside the cornerstone extension. Ideally nothing inside platform/* should rely on cornerstone

Reply:
Updated